infraestructure/aws/docs/frontend-group.md

```diff
@@ -98,3 +98,53 @@ Finally you can add users to this group, from which they will gain access to dep
 6. Select the *FrontendDeployment* Group and click in **Next: Review**
 7. Click on **Create User**
 8. Copy the link in the **Success** dialog and send it to the user, along with it's *Access key ID*, *Secret access key* and *Password*. Manage this information carefully, as it allows the access to the bucket
+
+## CloudFront Invalidations Policy
+
+This policy will grant permissions to invalidate CloudFront's CDN content.
+
+1. Access the [CloudFront console](https://console.aws.amazon.com/cloudfront/)
+2. Choose the ID of the distribution you want to create the invalidation policy for
+3. Copy the displayed **ARN** value. It should look something like: `arn:aws:cloudfront::some-id:distribution/distribution-id`
+4. Now, access the [IAM Service Dashboard](https://console.aws.amazon.com/iam)
+5. Click on **Policies** on the left side panel
+6. Click on **Create Policy**
+7. Click on **Select** button in the *Create Your Own Policy* option
+8. **Policy Name**: "frontend-cdn"
+9. **Description**: Invalidation Policy for Frontend CDN
+10. **Policy Document**:
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "acm:ListCertificates",
+                "cloudfront:GetDistribution",
+                "cloudfront:GetDistributionConfig",
+                "cloudfront:ListDistributions",
+                "cloudfront:ListCloudFrontOriginAccessIdentities",
+                "cloudfront:CreateInvalidation",
+                "cloudfront:GetInvalidation",
+                "cloudfront:ListInvalidations",
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "iam:ListServerCertificates",
+                "sns:ListSubscriptionsByTopic",
+                "sns:ListTopics",
+                "waf:GetWebACL",
+                "waf:ListWebACLs"
+            ],
+            "Resource": "ARN value copied in 3"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListAllMyBuckets"
+            ],
+            "Resource": "arn:aws:s3:::*"
```

Comment:
@mdesanti 
I removed bucket rule as it had no effect over the invalidation. Added a link that shows cloudfront can't be restricted to action-level permissions also.